web-common/src/features/connectors/explorer/TableMenuItems.svelte

```diff
@@ -159,7 +159,7 @@
   <NavigationMenuItem on:click={handleGenerateMetricsAndExplore}>
     <ExploreIcon slot="icon" />
     <div class="flex gap-x-2 items-center">
-      Generate an Explore dashboard
+      Generate metrics
```

Comment:
Currently, to be consistent with the menu options for the OLAP connectors:
1. This option should read "Generate dashboard" (because it generates an Explore file)
2. We should add another entry that only generates a Model and Metrics View file. No Explore file. It should be named "Generate metrics".

In the future, when our Metrics View files also generate Explore resources, then we can drop the "Generate dashboard" option.

Reply:
resolved